env/production/config.json

```diff
@@ -0,0 +1,3 @@
+{
+  "env": "stg"
```

Comment:
What is this for? And why is it stg?

Reply:
My bad, should be production, this provisions the environment on GitHub